src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php

```diff
@@ -98,7 +98,7 @@ protected function getCapabilitiesNode()
                 ->scalarNode('platform')->defaultValue('ANY')->end()
                 ->scalarNode('browserVersion')->defaultValue('9')->end()
                 ->scalarNode('browser')->defaultValue('firefox')->end()
-                ->scalarNode('ignoreZoomSetting')->defaultValue('false')->end()
+                ->booleanNode('ignoreZoomSetting')->defaultFalse()->end()
```

Comment:
Some libraries require true/false as string to function. I guess for that reason this wasn't noticed for so long.

Thanks.